tests/e2e/cucumber/features/integrations/reshare.feature

```diff
@@ -48,4 +43,14 @@ Feature: reshare
     And "Carol" reshares the following resource
       | resource         | recipient | type | role   |
       | folder_to_shared | Alice     | user | viewer |
-    And "Carol" logs out
+    Then "Alice" should see the following recipients
+      | resource         | recipient | role   | type  |
+      | folder_to_shared | Brian     | editor | user  |
+      | folder_to_shared | sales     | viewer | group |
+    When "Brian" updates following sharee role
+      | resource         | recipient | role                    | type  |
+      | folder_to_shared | sales     | custom_permissions:read | group |
```

Comment:
could you look please why 
```
Then "Brian" should see the following recipients
      | resource         | recipient | role   | type  |
      | folder_to_shared | sales     | Custom permissions | group |
```
doesn't work

Reply:
I have implemented this step in a way that whenever we need to set custom permissions then the permissions should be provided as:
```
| role                          |
| custom_permissions:read,share |
```
or
```
| role                                 |
| custom_permissions:read,update,share |
```
or other combinations


What the following should do?
```
| role               |
| Custom permissions |
```

Reply:
I meant that: if user shares with normal role (viewer or editor) he can see the following recipients. but if user shares with custom permissions he cannot check following recipients:

```
 When "Alice" shares the following resource using the sidebar panel
      | resource         | recipient | type | role   |
      | folder_to_shared | Brian     | user | editor |
Then "Alice" should see the following recipients
      | resource         | recipient | role   | type  |
      | folder_to_shared | Brian     | editor | user  |

```
works, but if we use custom permissions:

 ```
When "Alice" shares the following resource using the sidebar panel
      | resource         | recipient | type | role   |
      | folder_to_shared | Brian     | user | custom_permissions:read |
Then "Alice" should see the following recipients
      | resource         | recipient | role   | type  |
      | folder_to_shared | Brian     | Custom permissions | user  |
```

or 

```
Then "Alice" should see the following recipients
      | resource         | recipient | role   | type  |
      | folder_to_shared | Brian     | custom_permissions:read | user  |
```

doesn't work.

Reply:
Ahhh, now I get it.
I will look at it. Thanks :heart: